protocol/otlp/metrics.go

```diff
@@ -327,6 +327,9 @@ func StringifyAnyValue(a *commonv1.AnyValue) string {
 	case *commonv1.AnyValue_StringValue:
 		v = a.GetStringValue()
 
+	case *commonv1.AnyValue_BytesValue:
+		v = string(a.GetBytesValue())
```

Comment:
We should ideally transform/escape any non-acceptable characters in the resulting string such that the string will be guaranteed to be utf-8 AND will pass our filters for dimension values.